plugins/woocommerce/client/blocks/assets/js/blocks/featured-items/with-featured-item.tsx

```diff
@@ -295,7 +296,13 @@ export const withFeaturedItem =
 									dangerouslySetInnerHTML={ {
 										__html:
 											category?.description ||
-											product?.short_description,
+											product?.short_description ||
+											( product?.description?.length > 0
+												? getLongTextPreview(
+														product.description,
+														400
+												  )
+												: '' ),
```

Comment:
Consider adding a comment explaining why we’re returning a 400 here. Ideally, link to the relevant PHP logic for context.

Something like this could make sense:

```
// Returning 400 to match the validation logic in PHP: see https://github.com/Kallyan01/woocommerce/blob/585311cd260a97010fbaca2a692766ff9694fef4/plugins/woocommerce/src/Blocks/BlockTypes/FeaturedProduct.php#L88-L89
```